forge-gui/res/cardsfolder/upcoming/koilos_roc.txt

```diff
@@ -0,0 +1,11 @@
+Name:Koilos Roc
+ManaCost:4 U
+Types:Creature Bird
+PT:3/3
+K:Flash
+K:Flying
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield, create a tapped Powerstone token. (It’s an artifact with “{T}: Add {C}. This mana can’t be spent to cast a nonartifact spell.”)
```

Comment:
same issues with curly apostrophes and quotes

Reply:
would you happen to know how i could fix notepad++ to do this>